nb/documentation/interactions_with_geometry/interactions_with_geometry.md

```diff
@@ -164,43 +164,47 @@ Remesh can be used to break up the segments of a shape allowing it to be distort
 shape.remesh(4, 2) first breaks segments longer than 4 and then breaks segments longer than 2.
 At each step the shape is retriangulated to preserve manifold structure.
 
+Once a shape is remeshed it can be twisted or bent about the origin.
+
 ```JavaScript
-const aRectangle = Box(5, 5, 15).view();
+Box(157, 20).ex(1).y(25)
+  .remesh(10, 8, 4)
+  .op(s => s.bend(25).and(s.outline()))
+  .gridView()
+  .md('A rectangle bent into a ring with a central radius of 25');
 ```
 
 ![Image](interactions_with_geometry.md.14.png)
 
+A rectangle bent into a ring with a central radius of 25
+
+---
+### Rotate
+A shape can be rotated about the origin using the .rx(), .ry(), and .rz() commands.
+These take the number of turns as an argument .rz(1/8) would rotate the shape by 1/8th of a rotation.
+Multiple turns can be provided, which will produce one result per turn.
```

Comment:
That's neat, I wasn't aware of that feature

Reply:
Yeah, the fly in the ointment is that there's no way to post-process on a per turn basis, but I think a parallel op might work.

e.g., s.rz(0/1, 1/2).op(color('blue'), color('red')) might work, except it needs something to distinguish between an op on the whole shape and an op on s.n(0), s.n(1), s.n(2).